OBAKitTests/Location/RegionsServiceTests.swift

```diff
@@ -68,10 +70,9 @@ class RegionsServiceTests: OBATestCase {
         stubRegions(dataLoader: dataLoader)
```

Comment:
Please update the function signature comment to `saved to disk` instead of `user defaults`.